tests/wpt/web-platform-tests/dom/nodes/Document-createEvent.html

```diff
@@ -151,4 +151,8 @@
     }, 'Should throw NOT_SUPPORTED_ERR for pluralized non-legacy event interface "' + eventInterface + 's"');
   }
 });
+
+test(function() {
+  assert_equals(document.createEvent('storageevent'), "");
+}, "document.createEvent('storageevent') does not propagate the document's URL.")
```

Comment:
(given the text at https://dom.spec.whatwg.org/#concept-event-create and https://dom.spec.whatwg.org/#dom-document-createevent)